packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -56,11 +56,9 @@ spec:
           - |
             [[ $REPLICA_NAME =~ -([0-9]+)$ ]] || exit 1
             RID=${BASH_REMATCH[1]}
-            echo -n $RID >/root/.celo/replica_id
-            echo "Generating private key for rid=$RID"
-            celotooljs.sh generate bip32 --mnemonic "$MNEMONIC" --accountType validator --index $RID > /root/.celo/pkey
+            echo -n $RID > /root/.celo/replica_id
             echo 'Generating address'
-            celotooljs.sh generate account-address --private-key `cat /root/.celo/pkey` > /root/.celo/address
+            celotooljs.sh generate account-address --mnemonic "$MNEMONIC" --accountType validator --index $RID > /root/.celo/address
```

Comment:
Does this command work? I think `account-address` doesn't accept the `--accountType` and `--index` flags, maybe we should just save the generated bip32 provate key as an env variable and still use this command the same way

Reply:
Thanks, I mistook that for the public key function